projects/core/styles/mixins/text.scss

```diff
@@ -3,128 +3,69 @@
     color: var(--tui-text-01);
 }
 
-//headings
+// headings
+
 @mixin text-h1() {
-    @include font_heading(50px, 56px);
```

Comment:
Seems like text mixins are pretty useless now. @MarsiBarsi @fynnfeldpausch they could probably be used by people already, so we cannot drop them, right? How about we mark them all as deprecated and drop SCSS ones since they are a new addition?

Reply:
Yeah, that's why I did not remove them because I figured that they might be in use. The same applies to the `.font` mixin, which is also unused now and could be deprecated.

Reply:
I don't know any way other than comment. We wanted to release Angular stuff so much we didn't stop to look that a lot of global styles and mixins are pointless :) People would still use some layout library because built in Taiga UI classes are not powerful enough. I think we should remove all of this in 3.0 to not confuse people.

Reply:
Added deprecation comments to all mixins and also removed them from the docs page as deprecated code should not be actively included in the docs IMO.